packages/redux/src/raffles/actions/factories/fetchRafflesFactory.ts

```diff
@@ -0,0 +1,49 @@
+import * as actionTypes from '../../actionTypes';
+import {
+  Config,
+  GetRaffles,
+  Raffles,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import { normalize } from 'normalizr';
+import raffleSchema from '../../../entities/schemas/raffles';
+import type { Dispatch } from 'redux';
+import type { RafflesQuery } from '@farfetch/blackout-client/src/raffles/types/rafflesQuery.types';
+
+/**
+ * Method responsible for obtaining the raffles.
+ *
+ * @param getRaffles - Get raffles client.
+ *
+ * @returns Thunk factory.
+ */
+
```

Comment:
Not sure if JsDoc is valid with a space between the comment and the function 🤔

Reply:
space removed